hazelcast/src/test/java/com/hazelcast/internal/metrics/jmx/MemberJmxMetricsTest.java

```diff
@@ -63,4 +89,50 @@ public void testNoMBeanLeak() {
         member.shutdown();
         helper.assertNoMBeans();
     }
+
+    @Test
+    public void testMetricsNotExposedOverJmxIfSysPropIsNotSet() throws Exception {
```

Comment:
Since the JMX system property is not cleared in the other test methods, I think this test can see it as `true`?

Reply:
Good catch @vojtechtoman :+1: Added clearing the sysprop to `@Before`.